recipes-containers/docker/docker-ce_git.bbappend

```diff
@@ -0,0 +1,9 @@
+RRECOMMENDS_${PN} += " \
+    kernel-module-aufs \
```

Comment:
@Asiderr Is building of those modules enabled already in the meta-virtualization kernel config fragment (docker.cfg)?